.github/workflows/build_test.yaml

```diff
@@ -12,7 +12,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
I guess it is not required to switch back to 22.04 as the issues fixed

Reply:
Updated PR to set ubuntu-22.04 only.
~2 weeks  ago github set ununtu-latest to ubuntu-24, previously it was ubuntu 22.04